src/simulator.rs

```diff
@@ -0,0 +1,81 @@
+use cosmwasm_std::{Env, HexBinary};
+use sha2::{Digest, Sha256};
+
+/// Creates a predictable randomness seed
+///
+/// This generates a seed based on sha256 hash of the block height. to be used as a predictable randomness seed.
+/// Warning!! If you need an unpredictalble randomness do not use this function.
+/// Warning!! This function is intended for predictable randomness usecases. Or for a testnet environment.
+///
+/// ## Examples
+///
+/// Get a predictable coinflip using the randomness simulator:
+///
+/// ```
+/// use nois::{coinflip, Side,randomness_simulator};
+/// use cosmwasm_std::testing::mock_env;
+///
+/// let env = mock_env();
+/// let result = coinflip(randomness_simulator(env).to_array().unwrap());
+/// assert_eq!(result, Side::Tails);
+///
+/// ```
+///
+pub fn randomness_simulator(env: Env) -> HexBinary {
```

Comment:
Also let's take a reference here such that the user does not need to clone env:

```suggestion
pub fn randomness_simulator(env: &Env) -> [u8; 32] {
```